Sniffo/Sniffo.Core/IpHandler.cs

```diff
@@ -44,7 +44,15 @@ public IPAddress[] GetAllAvaliableIp()
 
         public string SniffedPackage()
         {
-            return IpV4Package();
+            return IPAddress?.AddressFamily switch
```

Comment:
The nullable for IPAddress must be validated before, here the property cannot be nullable, otherwise is going to throw the ```NotImplementedException```. And throwing this exception for this nullable is not user-friendly.